internal/databases/mongo/binary/dto.go

```diff
@@ -0,0 +1,29 @@
+package binary
+
+import (
+	"os"
+)
+
+type BackupDirectoryMeta struct {
+	Path     string      `json:"Path"`
+	FileMode os.FileMode `json:"FileMode"`
+}
+
+type Checksum struct {
+	Algorithm string `json:"Algorithm"`
+	Data      string `json:"Data"`
+}
+
+type BackupFileMeta struct {
+	Path             string      `json:"Path"`
+	FileMode         os.FileMode `json:"FileMode"`
+	Compression      string      `json:"Compression"`
+	CompressedSize   int64       `json:"CompressedSize"`
+	UncompressedSize int64       `json:"UncompressedSize"`
+	Checksum         Checksum    `json:"Checksum"`
+}
+
+type MongodBackupFilesMetadata struct {
+	BackupDirectories []*BackupDirectoryMeta `json:"BackupDirectories,omitempty"`
+	BackupFiles       []*BackupFileMeta      `json:"BackupFiles,omitempty"`
```

Comment:
Why not store it like this since we convert them to maps in the BackupService anyway?

```
	BackupDirectoriesMap map[string]*BackupDirectoryMeta
	BackupFilesMap       map[string]*BackupFileMeta
```

Reply:
Also, is it really necessary to divide the directories and files? Also, what about symlinks, etc? Why not store everything in the same place? We have a FileMode anyway.

Reply:
On the other hand, if we plan to store the backup files in TARs in the future versions, this might be not so important since we will rewrite this code anyway.

Reply:
> Why not store it like this since we convert them to maps in the BackupService anyway?

map needed to detect "visited" situation and get visited "value" during backup stage only. key of map is path, that already exists in value (so this redundant information in meta files).
map is small overhead.

> Also, is it really necessary to divide the directories and files? Also, what about symlinks, etc? Why not store everything in the same place? We have a FileMode anyway.

This is for convenience, because of I process files and directories separately. and files meta information contains more fields. and I see so this life =) or it is essential problem? 

> On the other hand, if we plan to store the backup files in TARs in future versions, this might be not so important since we will rewrite this code anyway.

I don't know =)